disperser/batcher/encoded_blob_store.go

```diff
@@ -102,7 +102,6 @@ func (e *encodedBlobStore) PutEncodingResult(result *EncodingResult) error {
 	}
 	e.encoded[requestID] = result
 	delete(e.requested, requestID)
-	e.logger.Trace("[PutEncodingResult]", "referenceBlockNumber", result.ReferenceBlockNumber, "requestID", requestID, "encodedSize", e.encodedResultSize)
```

Comment:
Reason for removing? Is it too noisy / not helpful?

Reply:
I included this to debug an issue but it's very noisy